manifests/http_port.pp

```diff
@@ -1,24 +1,43 @@
 define squid::http_port (
-  Variant[Pattern[/\d+/], Integer]
-          $port    = $title,
-  Boolean $ssl     = false,
-  String  $options = '',
-  String  $order   = '05',
+  Optional[Integer] $port    = undef,
```

Comment:
can you use this type? https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/port.pp#L1. Please then also bump the minimal stdlib version in metadata.json to 4.25.0.